uber/config.py

```diff
@@ -195,6 +195,11 @@ def PAGE(self):
         return cherrypy.request.path_info.split('/')[-1]
 
     @property
```

Comment:
Well, it's odd that it didn't just error, but actually gave me the model's full_name property. At any rate, I've fixed it.